src/test/java/racingcar/domain/CarsTest.java

```diff
@@ -0,0 +1,101 @@
+package racingcar.domain;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import racingcar.domain.Movable;
+import racingcar.domain.model.Car;
+import racingcar.domain.model.Cars;
+
+public class CarsTest {
+    private Cars cars;
+
+    @BeforeEach
+    public void InitCars() {
+        Car pobi = new Car("pobi");
+        Car woni = new Car("woni");
+        Car jun = new Car("jun");
+
+        pobi.moveForward(new Movable() {
+            @Override
+            public boolean canMove() {
+                return true;
+            }
+        });
+
+        pobi.moveForward(new Movable() {
+            @Override
+            public boolean canMove() {
+                return true;
+            }
+        });
+
+        woni.moveForward(new Movable() {
+            @Override
+            public boolean canMove() {
+                return true;
+            }
+        });
+
+        woni.moveForward(new Movable() {
+            @Override
+            public boolean canMove() {
+                return true;
+            }
+        });
+
+        jun.moveForward(new Movable() {
+            @Override
+            public boolean canMove() {
+                return true;
+            }
+        });
```

Comment:
중복이 너무 많습니다
아래와 같은 방법은 어떨까요?

```suggestion
// given
Movable movable = new Movable() {
    @Override
    public boolean canMove() {
        return true;
    }
}

// when
pobi.moveForward(movable);
woni.moveForward(movable);
```

Reply:
훨씬 간결해지네요..

좋은 지적 감사합니다!